pex/bin/pex.py

```diff
@@ -819,6 +819,7 @@ def do_main(
             compress=options.compress,
         )
         if options.seed != Seed.NONE:
+            pex = PEX(pex_file, interpreter=interpreter)
```

Comment:
Thanks for spotting, yeah it turned out unrelated. Eventually I figured out is was because my rendition of Distribution.location needed to realpath to be faithful to pkg_resource.Distribution. I never circled back to remove this. Gone.